cl/assets/react/TagList.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { usePaginatedQuery } from 'react-query';
+import TagListInner from './TagListInner';
+import { appFetch } from './_fetch';
+import { Tag, UserState } from './_types';
+
+
+const TagList: React.FC<UserState> = ({ userId, userName, isPageOwner }) => {
+  const [page, setPage] = React.useState(1);
+
+  const getTags = React.useCallback(
+    async (key: string, page = 1) =>
+      await appFetch(`/api/rest/v3/tags/?user=${userId}&page=${page}&page_size=10&order_by=name`),
+    []
+  );
+
+  const { isLoading, isError, error, resolvedData, latestData, isFetching } = usePaginatedQuery( // this has been deprecated...
```

Comment:
That's too bad. Best time to update it is now though. Have you looked at how hard it is?

Reply:
Shouldn't be too hard to adopt the new api. See, https://react-query.tanstack.com/guides/paginated-queries. It looks like the only thing extra that's needed is to pass the "keepPreviousData" option to true.

Reply:
This should be resolved.  thanks for the tip @litewarp